RELEASES.md

```diff
@@ -66,6 +66,9 @@ Unreleased.
   instead of returning an error.
   [#6776](https://github.com/bytecodealliance/wasmtime/pull/6776)
 
+* Optimized the cranlift compilation on aarch64
```

Comment:
*cranelift

Also I think it would make sense to say something about this happening only for large wasm module.